prosessering-jdbc/src/test/resources/db/migration/V1__schema.sql

```diff
@@ -2,7 +2,7 @@
 CREATE TABLE IF NOT EXISTS task (
     id            BIGSERIAL PRIMARY KEY,
     payload       VARCHAR                                              NOT NULL,
-    status        VARCHAR(15)  DEFAULT 'UBEHANDLET'::CHARACTER VARYING NOT NULL,
+    status        VARCHAR(20)  DEFAULT 'UBEHANDLET'::CHARACTER VARYING NOT NULL,
```

Comment:
viktig å dokumentere schema endringer godt i changeloggen slik at vi får med oss slike endringer i apper som skal ta ibruk denne versjonen.

Reply:
Denne endring har ikke noe å gjøre med selve versjonen, 15 er for litet for å bruke status MANUELL_OPPFØLGNING. Så er vel ett tegn på att ikke noen bruker MANUELL OPPFØLGNING ettersom det hadde feilet i de applikasjonene.
Kan selvfølgelig informere om det, man finner fort ut det i en integrasjonstest :)

Reply:
22169e7
Jeg la till det til readmen, egentligen burde disse kunne være uten størrelse i postgres